spec/controllers/queue_items_controller_spec.rb

```diff
@@ -82,11 +89,71 @@
         expect(user1.queue_items.size).to eq(0)
       end
 
-      it "redirect to queue index" do
+      it "should redirect to queue index" do
```

Comment:
The word `should` should be deleted not added. Its not needed - read more about this http://betterspecs.org/#should